### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by kryvets-vadym**
[DEMO LINK](https://kryvets-vadym.github.io/react_clock/)

### CODE REVIEW

File: src/Clock.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const today = new Date();
+
+      this.setState({ today });
+
+      // eslint-disable-next-line no-console
+      console.info(today.toUTCString().slice(-12, -4));
```
Comment (bohdan-stepanets) [inline src/Clock.tsx:25]: It'll be cool to make function that makes string from Date, use it here and at render (line 55)

File: src/App.tsx [hunk 2]
```diff
@@ -1,39 +1,69 @@
...
+  componentDidMount() {
+    document.addEventListener('click', this.handleLeftClick);
+    document.addEventListener('contextmenu', this.handleRightClick);
 
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
+    this.clockId = window.setInterval(() => {
+      this.setState({ clockName: getRandomName() });
```
Comment (bohdan-stepanets) [inline src/App.tsx:32]: Make a function for `this.setState({ clockName: getRandomName() })`

**Review by bohdan-stepanets — APPROVED** [review]
Good job!

**Review by Olexandr6 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**